src/Wellcome.Dds/Wellcome.Dds.AssetDomainRepositories/Storage/WellcomeStorageService/ArchiveStorageServiceWorkStore.cs

```diff
@@ -37,24 +40,26 @@ public class ArchiveStorageServiceWorkStore : IWorkStore
             }
 
             xmlElementCache = elementCache;
-            Identifier = identifier;
+            Identifier = identifier;            
+            StorageType = storageType;
             this.storageServiceClient = storageServiceClient;
             ArchiveStorageMap = archiveStorageMap;
             this.storageServiceS3 = storageServiceS3;
         }
 
-        public string Identifier { get; }
+        public string Identifier { get; }        
+        public string StorageType { get; }
 
         public string GetAwsKey(string relativePath)
-        {
-            const string awsKeyTemplate = "digitised/{0}/{1}/data/{2}";
+        {            
+            const string awsKeyTemplate = "{0}/{1}/{2}/data/{3}";
```

Comment:
Could named identifiers work here, rather than default string format? Likely less performant but more readable (might nto be worth tradeoff).

```cs
const string awsKeyTemplate = "{type}/{id}/{version}/data/{path}";
return awsKeyTemplate
  .Replace("{type}", StorageType)
  .Replace("{id}", Identifier)
  .Replace("{version}", versionSet.Key)
  .Replace("{path}", relativePath);
```